### PR TITLE
Show photos and videos in app #251

### PR DESCRIPTION
**Pull request by florianhaar**
**Link to the given issue**
https://github.com/open-xchange/ox-coi/issues/251

**Describe what the problem was / what the new feature is**
The method call to get the next media was missing.

**Describe your solution**
Added the missing method:

`dcContext.getNextMedia(messageId, dir, messageTypeOne, messageTypeTwo, messageTypeThree);`

Also added new arguments:
```
static final String ARGUMENT_DIR = "dir";
static final String ARGUMENT_MESSAGE_TYPE_ONE = "messageTypeOne";
static final String ARGUMENT_MESSAGE_TYPE_TWO = "messageTypeTwo";
static final String ARGUMENT_MESSAGE_TYPE_THREE = "messageTypeThree";
```

**Additional context**

**The iOS part is missing and needs to be added before merge.**

### CODE REVIEW

File: ios/Classes/Extensions/Flutter/FlutterMethodCall+Extensions.swift [hunk 1]
```diff
@@ -82,13 +82,13 @@ extension FlutterMethodCall {
...
     // MARK: - Parameter Values
-
+    
     func intValue(for key: String, defaultValue: Int32 = 0, result: FlutterResult) -> Int32 {
         guard let parameters = parameters else {
             return defaultValue
         }
 
-        if !contains(keys: [key]) {
+        if !contains(key: key) {
```
Comment (Boehrsi) [inline ios/Classes/Extensions/Flutter/FlutterMethodCall+Extensions.swift:91]: @phranck was this change intentional? Looks somehow strange, as nothing else was renamed.

**Review by Boehrsi — COMMENTED** [review]
Just one question everything else looks good.

File: ios/Classes/Extensions/Flutter/FlutterMethodCall+Extensions.swift (hunk 1 above)
Comment (ghost) [reply ios/Classes/Extensions/Flutter/FlutterMethodCall+Extensions.swift:91]: Yes, it has been changed by intention. A request of looking for an array of keys is never used but looking for one single key.

**Review by ghost — APPROVED** [review]
Looks goot to me.

**Review by Boehrsi — APPROVED** [review]
LGTM
